sematic/BUILD

```diff
@@ -80,6 +80,7 @@ sematic_py_lib(
         "//sematic/db/models:run",
```

Comment:
Will there be a follow-up PR with rendering the jobs' lifecycles in the external resource tab?

Reply:
Yes